### PR TITLE
[TW-1261] Delete outdated file, and fix links

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
The `adding-private-network.md` file was relocated to the `/private-api-network/` folder.

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Add elements to the Private API Network to help your team discover and use your
 
 ## Organizing with folders
 
-The sidebar navigation displays the folder structure for your Private API Network. API Network Managers can drag elements and sub-folders into different folders. You can add descriptions to folders to describe elements within the folders. You can also add Folder Managers to specific folders. To learn more about managing folders, see [Managing folders and elements](#managing-folders-and-elements).
+The sidebar navigation displays the folder structure for your Private API Network. API Network Managers can drag elements and sub-folders into different folders. You can add descriptions to folders to describe elements within the folders. You can also add Folder Managers to specific folders. To learn more about managing folders, see [Managing folders and elements](/docs/collaborating-in-postman/private-api-network/managing-private-network/#moving-folders-and-elements).
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md:57]: The link goes to "Moving folders and elements" heading, not "Managing..."

**Review by mark-anthony-postman — APPROVED** [review]
Do you need to add a redirect for the file you deleted (/docs/collaborating-in-postman/adding-private-network)?

Otherwise LGTM!

File: src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md:57]: Good question! Redirects should already be in place for the new file from this PR: https://github.com/postmanlabs/postman-docs/pull/4910

**Comment by MatthewPinsker-postman** [comment]
> Do you need to add a redirect for the file you deleted (/docs/collaborating-in-postman/adding-private-network)?
> 
> Otherwise LGTM!

Good question! Redirects should already be in place for the new file from this PR: https://github.com/postmanlabs/postman-docs/pull/4910

File: src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/private-api-network/organizing-private-network.md]: Thanks for catching this!

**Pull request merged**